src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    <p className="Person__partner">
+      {!person.isMarried && 'I am not married'}
+      {
+        person.isMarried && person.sex === 'm' ? `${person.partnerName} is my wife` : `${person.partnerName} is my husband`
+      }
```

Comment:
```suggestion
        {person.isMarried
          ? `${person.partnerName} is my ${partner}`
          : 'I am not married'
        }
```

Where partner is:

`const partner = person.sex === 'm' : 'wife' : ''husband;`